grafana-plugin/src/models/schedule/schedule.types.ts

```diff
@@ -7,6 +7,7 @@ export enum ScheduleType {
   'Calendar',
   'Ical',
   'API',
+  'all'
```

Comment:
technically `all` is not a schedule type, it's a union of other types, so it would be better not to include it